pages/schedule.tsx

```diff
@@ -85,6 +90,18 @@ export default function Schedule({ filters, shifts }: ISchedulesProps) {
     formatEvents();
   }, [selectedFilters, formatEvents]);
 
+  function getStorageFilters() {
+    const storageFilters = localStorage.getItem("storageFilters");
+    if (storageFilters) {
+      setSelectedFilters(JSON.parse(storageFilters));
+      // TODO: enable selected checkboxes
```

Comment:
I would like to see this feature implemented. What are your thoughts on it?

Reply:
Yes it is a requirement. Otherwise this feature would be a downgrade as far as UX, as it would be really unintuitive to figure out which filters are actually enabled